pyproject.toml

```diff
@@ -97,7 +97,7 @@ simplejson = "^3.17.0"
 SPARQLWrapper = "^1.8.5"
 SQLAlchemy = "1.4.41"  # latest stable version we can take - Will Jan 13 2023
 structlog = ">=19.2.0,<20"
-submit4dn = "0.9.7"
+submit4dn = "0.9.7"  # XXX: this version is likely not 3.11 compliant but will allow locking for now
```

Comment:
Does fourfront even use submit4dn?  Maybe this dependency should be removed or if it is used checked and updated as we are now on v4.0.0 in pypi.

Reply:
@willronchetti @aschroed rebuilding FF without submit4dn worked. I have updated mastertest and hotseat.